examples/validate.py

```diff
@@ -66,15 +66,15 @@ def loadAllJsonObjects(dir):
        
   return objects, badFiles
    
-def loadSchemas():
-  schemaTuples, badSchemaFiles = loadAllJsonObjects("schemas")
-  schemas = {}
-  for path, fileName, o in schemaTuples:
-    schemaName = os.path.basename(os.path.dirname(path))
-    versionName = fileName[:-5]
-    schemas[schemaName + "-" + versionName] = o
+def loadValidators():
+  schemas, badSchemaFiles = loadAllJsonObjects("schemas")
+  validators = {}
+  for path, fileName, o in schemas:
+    key = os.path.basename(os.path.dirname(path)) + "-" + fileName[:-5]
```

Comment:
Could we get back some names, like this?
```suggestion
   schemaName = os.path.basename(os.path.dirname(path))
   versionName = fileName[:-5]
   key = schemaName + "-" + versionName
```

The names make it easier to read what is happening.